keras/layers/core/einsum_dense.py

```diff
@@ -197,6 +197,10 @@ def kernel(self):
             )
         return self._kernel
 
+    @kernel.setter
+    def kernel(self, value):
+        self._kernel = value
```

Comment:
If we're going to do this, should we typecheck `kernel` to make sure it's a Variable? Also, we should definitely untrack the previously tracked `_kernel` variable otherwise it's still listed in `weights`.